.travis.yml

```diff
@@ -1,11 +1,42 @@
-language: minimal
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      env:
+        - ARCH=ppc64le
+
+    - os: linux
+      env:
+        - ARCH=aarch64
+
+    - os: linux
+      env:
+        - ARCH=amd64
```

Comment:
I don't think we should have this build. There was discussion that upstream wouldn't be happy, and their help is greatly appreciated

Reply:
ok